doc/man3/OSSL_HTTP_REQ_CTX.pod

```diff
@@ -90,8 +92,8 @@ encoding of I<req>, using the ASN.1 template I<it> to do the encoding.  The
 HTTP header C<Content-Length> is automatically filled out, and if
 I<content_type> isn't NULL, the HTTP header C<Content-Type> is also added with
 its content as value.  All of this ends up in the internal memory B<BIO>.
-This requires that the request type be C<POST>, i.e. that I<method_GET> is 0
-in the OSSL_HTTP_REQ_CTX_new() call.
+This requires that the request type be C<POST>,
+i.e., that I<method_POST> is 1 in the OSSL_HTTP_REQ_CTX_new() call.
```

Comment:
Please remove the comma after `i.e.`

Reply:
> Please remove the comma after i.e.

That's generally the wrong thing to do.   Perhaps avoid the issue:
```
This requires that the request type be C<POST> so that I<method_POST> is ...
```

Reply:
> > Please remove the comma after i.e.
> 
> That's generally the wrong thing to do.

What you you mean is the wrong thing?

> Perhaps avoid the issue:
> ```
> This requires that the request type be C<POST> so that I<method_POST> is ...
> ```
Though this looks like a way to avoid the AE vs. BE issue,
doing this change would reverse the cause and the effect in the semantics of the sentence.

Reply:
We could simply say:
```
This requires that I<method_POST> was 1 in the OSSL_HTTP_REQ_CTX_new() call.
```

Reply:
By "wrong thing to do" I meant according to English grammar rules, i.e. and e.g. almost always have commas before and after.  Your proposed wording is better.  What is `the AE vs BE issue` ?

Reply:
> By "wrong thing to do" I meant according to English grammar rules, i.e. and e.g. almost always have commas before and after.

Thanks for clarifying.

> Your proposed wording is better. 

Fine, so I've used that.

> What is `the AE vs BE issue` ?

What I meant is 
* the question with of the two is preferred, to which I don't recall there is a clear answer for this project, and 
* dependent on the first point, which consequences this has in detail, also how to handle existing/legacy texts.

Reply:
And the abbreviations `AE` and `BE` ?

Reply:
**A**merican **E**nglish vs. **B**ritish **E**nglish; 
pretty commonly used abbreviations.

Reply:
> **A**merican **E**nglish vs. **B**ritish **E**nglish;
> pretty commonly used abbreviations.

Never seen them before; thanks!

Reply:
German is reputed (at least in Sweden) for littering with commas...  that AE is turning more germanic is a surprise to me :wink:

Reply:
That being said, the changed phrasing was a good work-around